Client/Frontend/Sync/SyncPairWordsViewController.swift

```diff
@@ -14,7 +14,8 @@ class SyncPairWordsViewController: SyncViewController {
     var scrollView: UIScrollView!
     var containerView: UIView!
     var codewordsView: SyncCodewordsView!
-    
+    private let seedByteLength = 32
```

Comment:
This can probably stick to being a constant on `BraveSyncAPI`

Reply:
Reverted the change

Fix at https://github.com/brave/brave-ios/pull/4220/commits/2c24efcf0732de14988fb61de174f8fe57b56614